{"name": "test_add_group[None : name : header4oZFpstXbc49pJ9dzn9 : footerSIT4PhD9C9uWF]", "status": "broken", "statusDetails": {"message": "AttributeError: module pytest has no attribute allure", "trace": "app = <fixture.application.Application object at 0x000001FFD791D2D0>, db = <fixture.db.DbFixture object at 0x000001FFD791D210>, json_groups = None : name : header4oZFpstXbc49pJ9dzn9 : footerSIT4PhD9C9uWF, check_ui = False\n\n    def test_add_group(app, db, json_groups, check_ui):\n        group = json_groups\n>       with pytest.allure.step('Given a group list'):\n\ntest\\test_add_group.py:8: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nname = 'allure'\n\n    def __getattr__(name: str) -> object:\n        if name == \"Instance\":\n            # The import emits a deprecation warning.\n            from _pytest.python import Instance\n    \n            return Instance\n>       raise AttributeError(f\"module {__name__} has no attribute {name}\")\nE       AttributeError: module pytest has no attribute allure\n\nenv\\lib\\site-packages\\pytest\\__init__.py:165: AttributeError"}, "parameters": [{"name": "json_groups", "value": "None : name : header4oZFpstXbc49pJ9dzn9 : footerSIT4PhD9C9uWF"}], "start": 1655751425110, "stop": 1655751425110, "uuid": "9976522d-d790-4b31-b5a1-32f62a77ce04", "historyId": "f5fad0782555a50cba558abb156bf72a", "testCaseId": "8e9b1d07ef6cdb67de3c4a3d82e10652", "fullName": "test.test_add_group#test_add_group", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_add_group"}, {"name": "host", "value": "WIN11-Big-PC"}, {"name": "thread", "value": "17256-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_add_group"}]}